plaso/containers/artifacts.py

```diff
@@ -71,58 +71,62 @@ class OperatingSystemArtifact(ArtifactAttributeContainer):
   """Operating system artifact attribute container.
 
   Attributes:
-    name (str): name, such as "Linux", "MacOS" or "Windows".
-    product (str): product name, such as "macOS Mojave" or "Windows XP".
-    version (str): version, such as "10.14.1" or "5.1".
+    family (str): operating system family name, such as "Linux", "MacOS"
+        or "Windows". This value is used to programmatically link a parser
+        preset to an operating system and therefore must be one of predefined
```

Comment:
I think this means to be "predefined values in definitions.OPERATING_SYSTEM_FAMILIES". If not, please clarify.

There are a few other places where this docstring is replicated, please fix those also.

Reply:
ack